public/index.html

```diff
@@ -0,0 +1,171 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/public/css/reset.css" rel="stylesheet" />
+    <link href="/public/css/style.css" rel="stylesheet" />
+    <link href="/public/css/components/header.css" rel="stylesheet" />
+    <link href="/public/css/components/features.css" rel="stylesheet" />
+    <link href="/public/css/components/feature-list.css" rel="stylesheet" />
+    <link href="/public/css/components/commerce-info.css" rel="stylesheet" />
+    <link href="/public/css/components/footer.css" rel="stylesheet" />
+    <link href="/public/css/utils.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header container">
+      <div class="header-content container-header">
```

Comment:
Separate from the roles is the idea of programming. Even HTML or CSS which is not a programming language(In general. I think these are programming languages). 

Adding multi-class names is a solution for CSS naming batting / [specifificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). But in this case, you don't need to add muti-class for these elements.

```.css
.header-content {
  flex-direction: column;
  justify-content: center;
  display: flex;
  row-gap: 16px;
}
...
  .container-header {
    max-width: 93%;
    min-height: 145px;
    margin: 0 auto;
    padding-right: 0.5rem;
    padding-left: 0.5rem;
  }
```

You can mix these CSS in a ONE class name. Too separate to find the selectors in HTML and CSS files when you have to fix something.

Reply:
@sharray2918 
I have addressed the point of joining  `header-content` and `container-header` together.
Thank you for pointing this out as it made me look into "Minification" in order to remove unnecessary characters and whitespace from CSS files to reduce file size. 
Although I am only addressing this one you have pointed out, I will be addressing all missing areas.